test.c

```diff
@@ -2243,7 +2243,7 @@ upgrade_message_fix(char *body, const size_t nread, const size_t nmsgs, ...) {
   va_list ap;
   size_t i;
   size_t off = 0;
- 
+
```

Comment:
I know it is boring, but let's do it without style fixes in not relevant parts ;)